lib/Echidna/Solidity.hs

```diff
@@ -24,10 +24,11 @@ import Data.Maybe                 (isNothing)
 import Data.Monoid                ((<>))
 import Data.Text                  (Text, isPrefixOf, pack, unpack)
 import Data.Text.Lens             (unpacked)
-import Data.Text.Read             (decimal, hexadecimal)
+import Data.Text.Read             (decimal, {-hexadecimal-})
```

Comment:
we can remove this if we aren't using it

Reply:
Fixed.